lms/templates/registration/activate_account_notice.html

```diff
@@ -1,14 +1,18 @@
-<%! from django.utils.translation import ugettext as _ %>
+<%page expression_filter="h"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import Text, HTML
+%>
 <div class="wrapper-msg urgency-high">
     <div class="msg">
         <div class="msg-content">
             <h2 class="title">${_("Thanks for Registering!")}</h2>
             <div class="copy">
-                <p class='activation-message'>${_(
+                <p class='activation-message'>${Text(_(
                         "You've successfully created an account on {platform_name}. We've sent an account "
-                        "activation message to {email}. To activate your account and start enrolling in "
+                        "activation message to {email_start}{email}{email_end}. To activate your account and start enrolling in "
                         "courses, click the link in the message."
-                        ).format(email="<strong>{}</strong>".format(email), platform_name=platform_name
+                        )).format(email_start=HTML("<strong>"), email_end=HTML("</strong>"), email=email, platform_name=platform_name
```

Comment:
Nit: for better readability, consider changing the indentation so the format parameters are on their own lines.